NitroxClient/GameLogic/Entities.cs

```diff
@@ -20,6 +21,7 @@ public class Entities
         private readonly DefaultEntitySpawner defaultEntitySpawner = new DefaultEntitySpawner();
         private readonly SerializedEntitySpawner serializedEntitySpawner = new SerializedEntitySpawner();
         private readonly Dictionary<TechType, IEntitySpawner> customSpawnersByTechType = new Dictionary<TechType, IEntitySpawner>();
+        private readonly Dictionary<NitroxModel.DataStructures.Int3, EntityCell> EntityCells = new Dictionary<NitroxModel.DataStructures.Int3, EntityCell>();
```

Comment:
Aren't entity cells also dictated by level and not just Int3?

Reply:
Added New class named CellRoot to differentiate EntityCells